UI/Pages/Cookies.cshtml.cs

```diff
@@ -7,15 +7,77 @@ namespace UI.Pages
     public class Cookies : PageModel
     {
         private const string ConsentCookieName = ".FindATuitionPartner.Consent";
-        public IActionResult OnGet(bool? consent, string returnUrl)
+        public bool? Consent { get; set; }
+        public bool PreferencesSet { get; set; } = false;
+        public string? returnPath { get; set; }
+
+        public IActionResult OnGet(bool? consent, string returnUrl, bool? viewCookieStatement)
         {
+            returnPath = returnUrl;
+
+            if (Request.Cookies.ContainsKey(ConsentCookieName))
+            {
+                Consent = bool.Parse(Request.Cookies[ConsentCookieName]);
+            }
```

Comment:
There are a couple of null reference warnings that I think would be good to resolve